client/components/jetpack/jetpack-presales-chat-widget/index.tsx

```diff
@@ -38,10 +44,16 @@ async function fetchWithRetry(
 	}
 }
 
-export const ZendeskJetpackChat: React.VFC = () => {
+export const ZendeskJetpackChat: React.VFC< { keyType: KeyType } > = ( { keyType } ) => {
 	const [ error, setError ] = useState( false );
 	const { data: isStaffed } = usePresalesAvailabilityQuery();
-	const zendeskChatKey = config( 'zendesk_presales_chat_key' ) as keyof ConfigData;
+	const zendeskChatKey = useMemo( () => {
+		return config(
+			keyType === 'jpAgency'
```

Comment:
Not a blocker, but maybe we can make `keyType` a string instead of union type here and pass the key name and avoid the need to map. This way, when we add a new key later, we do not need to modify this line of code.

But I understand why you did it here to prevent pulling config data, not necessarily a chat key.